database/raft/raft.go

```diff
@@ -510,7 +510,6 @@ func (sv *Service) get(ctx context.Context, key string) ([]byte, error) {
 			err := sv.Set(ctx, "/dummyWrite", []byte(""))
 			if err != nil {
 				// TODO(tessr): should probably return this error
```

Comment:
could you update this todo to "do something about this error"? 

or... not? we do occasionally get this error and it's totally fine, the node just retries the dummy write.